### ISSUE TITLE
NVDA Community Portal directs to Sourceforge no longer used as a download repository

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5370]
**Reported by bhavyashah on 2015-09-18 10:23**
_This is a trivial issue_
On the NVDA Community portal website, go to the end of the document by pressing Ctrl + End.
While up arrowing, you will observe 'Downloads hosted by: SourceForge.net logo'
This link should be removed as NVDA is longer hosted by Sourceforge.

**Comment by leonardder** [comment]
Trac specific, which we no longer use. Closing.
